### ISSUE TITLE
performance and loss of the frame-based model

### ISSUE THREAD

**Issue by fabianbosshard** [YuanGongND/ssast#32]
I also wanted to ask if you still have the `result.csv` from the frame-based base model trained on audioset2m/librispeech, i.e. `mask01-base-f128-t2-b24-lr1e-4-m400-pretrain_joint-asli`. If you still have it, we would be very thankful if you could upload it here. We are currently trying to reproduce the pretraining results, such that we can then build on that and finetune it for speaker verification.

Best Regards,
Fabian

**Comment by YuanGongND** [comment]
[result.csv](https://github.com/YuanGongND/ssast/files/14984887/result.csv)

It has been a long while, but I tried to find this in our server. 

We re-run experiment with cleaned up code and generate these logs, there might be a small difference between this and that reported in the paper.

-Yuan

**Comment by fabianbosshard** [comment]
Thank you very much for your help! We appreciate the time and effort you've put into this.

-Fabian
